proto/osmosis/incentives/query.proto

```diff
@@ -49,6 +49,13 @@ service Query {
     option (google.api.http).get =
         "/osmosis/incentives/v1beta1/upcoming_gauges";
   }
+
+  rpc UpcomingGaugesPerDenom(UpcomingGaugesPerDenomRequest)
```

Comment:
nit: add a godoc for this handler 👍

Reply:
Also, why is it structured like this? Makes it very hard to read. Did we not add the vscode proto formatting changes @ValarDragon?

Reply:
Theres no vscode proto formatting AFAIK. I'm down for us to add it. (IIRC you did it via clang, not sure how that install works across machines / operating systems)

Reply:
https://github.com/osmosis-labs/osmosis/pull/1221

:P

Reply:
ah cool! Make proto-format should fix this as well.

@xBalbinus can you run `make proto-format` and commit that change before we merge this in